tapir/shifts/views/calendars.py

```diff
@@ -136,12 +141,28 @@ def get_context_data(self, **kwargs):
 class ShiftTemplateGroupCalendar(LoginRequiredMixin, TemplateView):
     template_name = "shifts/shift_template_group_calendar.html"
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data()
-        date_to_group = {}
-        today = timezone.now().date()
-        for week in range(52):
-            monday = today - timedelta(days=today.weekday()) + timedelta(weeks=week)
-            date_to_group[monday] = get_week_group(monday).name
-        context["date_to_group"] = date_to_group
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+
+        # TODO Frederik: a static calendar.html would accelerate
+        thisyear = date.today().year
+        start_date = date(thisyear, 1, 1)
+        end_date = date(thisyear, 12, 31)
+        delta = end_date - start_date  # returns timedelta
+        shift_dict = {}
```

Comment:
Maybe rename to `week_group_name_by_week` or something really expressive to give a better idea what this actually does?